documentation/contributing/pullrequests/index.markdown

```diff
@@ -159,8 +157,16 @@ If each step ends with issues, they need to be fixed before moving on.
 1. Notify maintainers to resume new merge.
 1. Write release notes on moveit.ros.org (e.g. [1](https://github.com/ros-planning/moveit.ros.org/pull/115), [2](https://github.com/ros-planning/moveit.ros.org/pull/110)). Send it to [Discourse MoveIt category](https://discourse.ros.org/c/moveit).
 
-### Release versioning policy
-
-* We use the minor version to differentiate the releases from different development branches.
-  * As of May 2019, Kinetic 0.9.x (kinetic-devel), Melodic 1.0.x (melodic-devel)
-* See also a [discussion for the best practice for versioning](https://discourse.ros.org/t/maintainer-best-practices-handling-changes-through-ros-releases/771) on discourse.
+### Release Versioning Policy
+
+* **Major version number**:
+  * Until 2019, MoveIt was never officially "out of beta" so was 0.x.x version
+  * As ROS 2 became more of a thing, MoveIt for ROS 1 was announced to be "out of beta" and increased to 1.x.x
+  * MoveIt for ROS 2 (MoveIt 2) uses the major version number 2.x.x
+* **Minor version number:** differentiates different ROS distributions
+  * Indigo 0.8.x (indigo-devel)
+  * Kinetic 0.9.x (kinetic-devel)
+  * Melodic 1.0.x (melodic-devel)
+  * Noetic 1.1.x (noetic-devel)
+* Minor version numbers are increased *right after the previous release*, in advanced preparation of the upcoming release. See [rational](https://github.com/ros-planning/moveit/issues/2036).
```

Comment:
Agreed with @rhaschke

Reply:
Maybe nitpicking ... The heading of this section is ***minor*** version numbers, you can only increase those once after branching or you run into conflicts with other ros-distros. Incrementing the ***patch*** version number after each patch-release can be done, and I'm in no way against it, but shouldn't be strictly necessary due to API guarantees? (However, we follow that procedure internally as well and it works just fine)

Reply:
Restricting the early bump to **minor** version numbers (and not considering patch version numbers as well for this policy) would indeed simplify matters a lot because the non-standard and more difficult procedure described in #446 then only needs to be applied once in a while. Actually, I like this idea. I didn't thought of this restriction before.